x-pack/legacy/plugins/siem/public/plugin.tsx

```diff
@@ -27,21 +27,24 @@ import {
   TriggersAndActionsUIPublicPluginSetup,
   TriggersAndActionsUIPublicPluginStart,
 } from '../../../../plugins/triggers_actions_ui/public';
+import { SecurityPluginSetup } from '../../../../plugins/security/public';
 
 export { AppMountParameters, CoreSetup, CoreStart, PluginInitializerContext };
 
 export interface SetupPlugins {
   home: HomePublicPluginSetup;
-  usageCollection: UsageCollectionSetup;
+  security: SecurityPluginSetup;
   triggers_actions_ui: TriggersAndActionsUIPublicPluginSetup;
+  usageCollection: UsageCollectionSetup;
 }
 export interface StartPlugins {
   data: DataPublicPluginStart;
   embeddable: EmbeddableStart;
   inspector: InspectorStart;
   newsfeed?: NewsfeedStart;
-  uiActions: UiActionsStart;
+  security: SecurityPluginSetup;
```

Comment:
This should be the Start contract, `SecurityPluginStart`

Reply:
`SecurityPluginStart` does not have the `authc` we need. It is only in setup. thats why we needed to add it explicitly in the object https://github.com/elastic/kibana/blob/master/x-pack/plugins/security/public/plugin.tsx#L137

Reply:
Ohhh, I get it now. We're doing some similar stuff on the server, but the basic idea is: store dependencies in setup() to use later in start(). This should be fine for now, but I'll follow up with platform and change it in the NP migration if necessary. 👍